src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs

```diff
@@ -66,7 +102,13 @@ public override Stream Body
 
         public override HttpHeadersCollection Headers => _headers ??= new HttpHeadersCollection(_httpData.NullableHeaders.Select(h => new KeyValuePair<string, string>(h.Key, h.Value.Value)));
```

Comment:
Unrelated question: Should we consider brining the lazy loading behavior to Headers as well ?

Reply:
@fabiocav @brettsam